03-JavaServiceSkeleton/MessageService/app/src/main/java/com/example/shevchenko/message/service/MessageService.java

```diff
@@ -0,0 +1,40 @@
+package com.example.shevchenko.message.service;
+
+import android.app.Service;
+import android.content.Intent;
+import android.os.IBinder;
+import android.os.RemoteException;
+import android.util.Log;
+
+public class MessageService extends Service {
+    private static String message;
+    private final static String TAG = "MessageService";
+
+    public MessageService() {
+    }
+
+    private IMessageAidlInterface.Stub messageService = new IMessageAidlInterface.Stub() {
+        @Override
+        public void produceMessage(String text) throws RemoteException {
+            Log.e(TAG, "Produce message = " + text);
+            message = text;
+            Log.e(TAG, "New message value = " + text);
+        }
+
+        @Override
+        public String consumeMessage() throws RemoteException {
+            Log.e(TAG, "Consume message = " + message);
+            return message;
+        }
+    };
+
+    @Override
+    public IBinder onBind(Intent intent) {
+        return messageService;
+    }
+
+    @Override
+    public boolean onUnbind(Intent intent) {
```

Comment:
No need to overload the default behavior method

Reply:
Fixed in the commit 5f20cea483dbfaa98ab31de2ffd831bf3de60bd6